### DISCUSSION TITLE
Using Incremental Static Regeneration (ISR) with fallback: "blocking"

### DISCUSSION THREAD

**Discussion by HsnVahedi** [vercel/next.js#20093] [Help]
According to [Data Fetching Documentation](https://nextjs.org/docs/basic-features/data-fetching#getstaticprops-static-generation), Incremental Static Regeneration works well with "fallback: true".

Is it also ok to use ISR with "fallback: 'blocking'" ?

I've tested and found no problems. But in documents, only using ISR with "fallback: true" is mentioned.

**Comment by PsyGik** [comment]
I know I am late to this party but [the docs have updated around ISR](https://vercel.com/docs/next.js/incremental-static-regeneration)

![image](https://user-images.githubusercontent.com/5432911/122971247-770b5880-d3ac-11eb-9af6-3d41c6b39377.png)
